src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -434,6 +442,17 @@ default String pkWebhook() {
         return "";
     }
 
+    @ConfigItem(
+        keyName = "bankWebhook",
```

Comment:
as discussed, rename config key & variable name prefixes from `bank` to `groupStorage`